src/workerinfo.js

```diff
@@ -34,57 +35,65 @@ class WorkerInfo {
     }
   }
 
-  async seen(provisionerId, workerType) {
+  async seen(provisionerId, workerType, workerGroup, workerId) {
     const expired = entity => Date.now() - new Date(entity.expires) > 24 * 60 * 60 * 1000;
     const expires = taskcluster.fromNow('5 days');  // temporary hard coded expiration
```

Comment:
Much shorter than 1 day risks a worker expiring while running a long task.  Ideally at some point we'll call workers "seen" on reclaimWork, at which point we could reduce this to an hour or so, but not yet.